examples/RESTBase/IntegrationTestExamples.yaml

```diff
@@ -0,0 +1,30 @@
+# Examples are similar to:
+#    * https://github.com/wikimedia/restbase/blob/master/test/features/pagecontent/pagecontent.js#L72
+#    * https://github.com/wikimedia/restbase/blob/master/test/features/pagecontent/pagecontent.js#L56
+#
+# Example run: ./phester.php test https://en.wikipedia.beta.wmflabs.org/api/rest_v1/ examples/RESTBase/IntegrationTestExamples.yaml
+
+suite: RESTBase integration tests
+description: Integration tests
+type: REST
+
+tests:
+  - description: Should transparently create a new HTML revision with id 252937
```

Comment:
That assumes that a revision with that ID exists, right?

Reply:
Yes, Petr wanted to see what one of the tests would look like in Phester: https://github.com/wikimedia/restbase/blob/master/test/features/pagecontent/pagecontent.js#L72

Reply:
Yes. Most of RESTBase tests are actually integration tests, so RB is pointing to a certain domain of the wiki (for this test, en.wikipedia.beta.wmflabs.org) and we ensure that a certain revision exist. 

So this test will only work for beta enwiki restbase. For service-checker tests we point them to a specially created and protected page on enwiki.

I don't know about Pheser enough to understand if you can ensure the page is created.

Reply:
> I don't know about Pheser enough to understand if you can ensure the page is created.
It can, but that should only be done on a dummy wiki, not a live wiki.